src/components/icon/icons/svgs/Ellipsis/ellipsis.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable max-len */
+export default {
+  __html:
+    '<svg class="carbon-icon__svg carbon-icon__svg--ellipsis" width="16px" height="16px" viewBox="0 0 16 16">' +
```

Comment:
I'm guessing lint will fail on this line as the `+` should be at the start of the new line.